test/plots/scatterPlotTests.ts

```diff
@@ -34,8 +40,28 @@ describe("Plots", () => {
         assert.isDefined(plot.animator(Plottable.Plots.Animator.MAIN), "main animator is defined");
         assert.isDefined(plot.size(), "size() is defined");
         assert.isDefined(plot.symbol(), "symbol() is defined");
+      });
 
-        svg.remove();
+      it("initially draws the points with a size of 0 when resetting", () => {
+        const data = [
+          { x: 0, y: 0 }
+        ];
+        const dataset = new Plottable.Dataset(data);
+        plot.addDataset(dataset);
+
+        const symbolFactory = Plottable.SymbolFactories.square();
+        plot.symbol(() => symbolFactory);
+
+        plot.animator(Plottable.Plots.Animator.MAIN, new Mocks.NoOpAnimator());
+        plot.animated(true);
+
+        plot.renderTo(svg);
+
+        const pathString = plot.content().select("path").attr("d");
+        const zeroSizePathString = symbolFactory(0);
+        const normalizingPath = svg.append("path"); // generated and normalized paths don't align on all browsers
+        const expectedPathString = normalizingPath.attr("d", zeroSizePathString).attr("d");
```

Comment:
Yeah. I tried to use the normalization function in `TestMethods`, but it doesn't always work correctly due to different choices regarding repeating path commands.